source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRInputRenderer.mm

```diff
@@ -217,20 +218,30 @@ - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
         ACRAggregateTarget *target = [[ACRAggregateTarget alloc] initWithActionElement:acoAction rootView:rootView];
 
         switch (action->GetElementType()) {
-            case ActionType::ShowCard:
-                [button addTarget:target action:@selector(toggleVisibilityOfShowCard) forControlEvents:UIControlEventTouchUpInside];
-                break;
+
             case ActionType::Submit:
                 [txtInput setReturnKeyType:UIReturnKeySend];
                 quickReplyView.target = target;
```

Comment:
same: consistency issue here around the usage of curly braces (we should use them like you do in the new block of code you added); Is there an editorconfig/clang-formatting rule or something that we can enable to catch and auto-fix these syntax rules? @paulcam206?

Reply:
filed #3381 to track this ask

---
In reply to: [316344102](https://github.com/microsoft/AdaptiveCards/pull/3354#discussion_r316344102) [](ancestors = 316344102)